src/Utils/relayHelper.ts

```diff
@@ -0,0 +1,4 @@
+export type CleanRelayFragment<T> = Omit<
```

Comment:
**suggestion:** I would maybe rename this file to `relayHelper.ts`

Reply:
I'll move the helper to [typeSupport](https://github.com/artsy/force/blob/main/src/Utils/typeSupport.ts) like @MrSltun suggested.